content/sigs/chinese-localization/index.adoc

```diff
@@ -53,11 +53,19 @@ If you are interested in Chinese localization for Jenkins website, please fork l
 
 We will maintain all Simplified Chinese resource files in link:https://github.com/jenkinsci/localization-zh-cn-plugin[here] that include Jenkins core and all plugins.
 
+=== WeChat
+
+Jenkins WeChat subscription account will be a very important platform to share Jenkins events and news for Chinese users. You can find all details about it from link:https://github.com/jenkins-infra/jenkins.wechat[here]. Please scan the QR code below:
+
+image::/images/jenkins-wechat.png[Jenkins WeChat,129,float="center"]
```

Comment:
typo, '::' => ':'

Reply:
I tried that. But it'll cause an uncomfortable layout. The image could take a single line if it follows `::`.